test/parallel/test-os.js

```diff
@@ -136,28 +137,30 @@ switch (platform) {
     }
 }
 
-// assert cidr prop present all interface info
 function flatten(arr) {
-  const _flatten = function(p, c) {
+  function _flatten(p, c) {
     if (Array.isArray(c)) p = p.concat(c.reduce(_flatten, []));
     else p.push(c);
     return p;
-  };
+  }
 
   return arr.reduce(_flatten, []);
-};
+}
 const netmaskToCIDRSuffixMap = new Map(Object.entries({
   '255.0.0.0': 8,
   '255.255.255.0': 24,
   'ffff:ffff:ffff:ffff::': 64,
   'ffff:ffff:ffff:ffff:ffff:ffff:ffff:ffff': 128
 }));
-flatten(Object.values(interfaces)).forEach((v) => {
-  assert.ok('cidr' in v, '"cidr" prop not found in networkInterfaces() result');
-  if (netmaskToCIDRSuffixMap.has(v.netmask)) {
-    assert.strictEqual(v.cidr, `${v.address}/${netmaskToCIDRSuffixMap.get(v.netmask)}`);
-  }
-});
+// assert cidr prop present all interface info
+flatten(Object.values(interfaces))
+  .map((v) => {
```

Comment:
You don't have to care that the `map.get` returns `undefined`. It's not pure FP, but it's susinct
```js
flatten(Object.values(interfaces))
  .map((v) => ({v, mask: netmaskToCIDRSuffixMap.get(v.netmask)}))
  .forEach(({v, mask}) => {
    assert.ok('cidr' in v, `"cidr" prop not found in ${inspect(v)}`);
    if (mask)
      assert.strictEqual(v.cidr, `${v.address}/${mask)}`)
  });
```